### PR TITLE
Add square column option

### PR DESCRIPTION
**Pull request by leandrocfe**

### CODE REVIEW

**Comment by zepfietje** [comment]
@leandrocfe, isn't it already square by default?

**Comment by leandrocfe** [comment]
> @leandrocfe, isn't it already square by default?
Hi @zepfietje ! Take a look:

Normal:
image

Square:
image

**Comment by zepfietje** [comment]
Would you use both in the same project?

I feel like rounding the square image borders is an opinionated style you could add using CSS. After all, square is the default already. Thoughts?

**Comment by leandrocfe** [comment]
> Would you use both in the same project?
> 
> I feel like rounding the square image borders is an opinionated style you could add using CSS. After all, square is the default already. Thoughts?

@zepfietje The idea would be a new style that you can apply easily. But I agree with you.

**Comment by zepfietje** [comment]
Yeah I get the goal, but would you use both in the same project, @leandrocfe?

If not, I don't see why we should add this as it's basically just an opinionated style for the square image.

**Comment by zepfietje** [comment]
@leandrocfe, I misunderstood this PR, sorry.
Thought the image is square already, but it's actually rectangular (not necessarily square).

We can add `square()` but it shouldn't have rounded corners.

File: packages/tables/src/Columns/ImageColumn.php [hunk 1]
```diff
@@ -153,9 +170,14 @@ public function getWidth(): ?string
         return $width;
     }
 
-    public function isRounded(): bool
```
Comment (danharrin) [inline packages/tables/src/Columns/ImageColumn.php:156]: Please reintroduce this method & deprecate. Call `$this->isCircular()` internally. Also update docs if there are any?

**Comment by leandrocfe** [comment]
> @leandrocfe, I misunderstood this PR, sorry. Thought the image is square already, but it's actually rectangular (not necessarily square).
> 
> We can add `square()` but it shouldn't have rounded corners.

@zepfietje It makes sense. Do you think that shadow and rounded can be options as well?

feature

And the user can control other attributes with extra attributes... What is your opinion? I'm not sure...

**Comment by zepfietje** [comment]
I think shadow and rounding are too opinionated and specific to have dedicated properties for. We could add `filament-tables-image-column-circular` and `filament-tables-image-column-square` CSS class hooks to make it easier to style though.

**Comment by leandrocfe** [comment]
> I think shadow and rounding are too opinionated and specific to have dedicated properties for. We could add `filament-tables-image-column-circular` and `filament-tables-image-column-square` CSS class hooks to make it easier to style though.

Nice! I agree 👍

**Comment by zepfietje** [comment]
@danharrin this should be ready to go! 🚀

**Comment by danharrin** [comment]
Thank you very much @leandrocfe @zepfietje

**Pull request merged**